doc/designs/expired_certificate_pruning.md

```diff
@@ -236,8 +236,8 @@ Overview of the CLI commands. Example:
 
 | Command |	Options |
 | --- | ----- |
-| ipa-acme-manage pruning | --enable=TRUE |
-| ipa-acme-manage pruning | --enable=FALSE |
+| ipa-acme-manage pruning | --enable |
+| ipa-acme-manage pruning | --disable |
 | ipa-acme-manage pruning | --cron=`"0 0 1 * *"` |
```

Comment:
You might need to remove quotes from cron.

Reply:
The quotes are needed when running in a shell so that they are considered a single argument. When running a command within ipatests they are not needed because it is already sent as a single string.